content/en/users/Rucio/Getting_started/_index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Getting Started"
+type: docs
+linkTitle: "Rucio Getting started"
+weight: 100
```

Comment:
from the preview here https://docs.egi.eu/documentation/383/users/rucio/ , i would change the weights in order to have the menu in this order: 1 - Rucio Getting started , 2- Rucio commands, 3 -Rucio admins

Reply:
Addressed in latest push